llvm/docs/UserGuides.rst

```diff
@@ -102,6 +103,10 @@ Clang
 :doc:`CFIVerify`
   A description of the verification tool for Control Flow Integrity.
 
+:doc: `KeyInstructionsClang`
```

Comment:
Does this render correctly as a link to your page...? None of the other lines have a space between `:doc:` and the page name, but maybe it works...?